CHANGELOG-developer.next.asciidoc

```diff
@@ -52,6 +52,8 @@ The list below covers the major changes between 7.0.0-rc2 and master only.
 
 ==== Added
 
+- Add configuration for APM instrumentation and expose the tracer trough the Beat object. {pull}17938[17938]
+- Make the behavior of clientWorker and netClientWorker consistent when error is returned from publisher pipeline
```

Comment:
this changelog entry looks wrong

Reply:
good catch, this seems to come from https://github.com/elastic/beats/pull/14506/files
probably needs a fix in master